### ISSUE TITLE
Revit Lookup Cast Error Wpf.Ui.Controls.TitleBar

### ISSUE THREAD

**Issue by binghuinew** [jeremytammik/RevitLookup#272]
### RevitLookup version

[2025.0.8]

### Steps to reproduce

- open dynamo
- run a python node (for example: calculate 3+5)
- run Revit Lookup (can not)

### ✔️ Expected Behavior

Revit Lookup can be opened.

### ❌ Actual Behavior

When I open Dynamo and run a Python node (— where I’m only calculating 3+5), whether it’s Python or “Python from Script”, everything works as expected. However, after this, I am unable to open Revit Lookup. 

If I debug Revit Lookup, I got this wpf error: 
![Snipaste_2024-08-12_13-03-10](https://github.com/user-attachments/assets/e56c7226-66d8-42af-be02-b5fe6a99f740)

![Unbenanntes Bild](https://github.com/user-attachments/assets/bf3cdb1b-a48e-4842-98b6-04679ebe19c4)

on dynamo Forum: https://forum.dynamobim.com/t/revit-lookup-error-after-running-dynamo-python-node/103472

### Other Software

Dynamo Core: 3.0.3.7597
Dynamo Revit: 3.0.3.4605
Revit Version: 2025.1
Revit Lookup: [2025.0.8]

**Comment by Nice3point** [comment]
Bug confirmed. It is only observed when opening Python Script, which loads about 80 additional dlls into memory, including an additional RevitLookup.UI dependency. If Lookup is already open, the search field breaks and does not display properly.
There are non-obvious Python Script mechanics here that I'm not aware of, need more research and help from the Dynamo community

![изображение](https://github.com/user-attachments/assets/342d9aff-5f67-4010-8a22-50ec32fe23ef)

**Comment by planUnd** [comment]
Dear all, 
this issue was fixed in RevitLookup, however I would like to know how  this was fixed, unfortunately the merge includes quite a lot of other stuff and I couldn't identify the part solving the problem.

**Comment by Nice3point** [comment]
@planUnd we disabled custom isolation and implemented Repacking. After fixes on Autodesk side it will be enabled again. +- with Revit 2027 release

**Comment by planUnd** [comment]
@Nice3point   thanks for the quick replay, so you completely removed app isolation from the code and instead packaged all into a single dll to avoid dll conflicts? 

So ideas suggested here: https://github.com/Nice3point/RevitToolkit about app Isolation where dropped?

Is there any documentation on why app isolation isn't working? is this true for Revit 2025 and 2026.

Did you ever get to the bottom why this double loading happens when executing a phyton node in dynamo?

**Comment by Nice3point** [comment]
>  thanks for the quick replay, so you completely removed app isolation from the code and instead packaged all into a single dll to avoid dll conflicts?

correct

> So ideas suggested here: https://github.com/Nice3point/RevitToolkit about app Isolation where dropped?

yes, to avoid conflicts with Revit builtin isolation. https://github.com/Nice3point/RevitToolkit/releases/tag/2026.0.0

> Is there any documentation on why app isolation isn't working? is this true for Revit 2025 and 2026.

this thread: https://feedback.autodesk.com/project/forum/thread.html?cap=cb0fd5af18bb49b791dfa3f5efc47a72&forid=%7B057e532f-e478-43d9-affc-01b3deb82a76%7D&topid=%7B8C202188-9EA5-49BE-B95F-7F5115507C88%7D&tp=3#toppost

> Did you ever get to the bottom why this double loading happens when executing a phyton node in dynamo?

dont know, something Dynamo code related

**Comment by planUnd** [comment]
@Nice3point 
So our issue ist in Revit 2025.4 and we based our App Isolation on the ideas propose in revitToolkit and simplified it for our needs. 

We got to "a" bottom. It seams that as soon as you start a phyton plugin. The python runtime  starts to go through all loaded assemblies and  calls the methode Assembly  a.GetExportetTypes() so that you can access the commands within python. 

And this somehow loads dependencies again, resulting into issues when e.g. loading a initializing a WPF component... 

So bottom line is app isolation doesn't work in revit 2025 as long nobody calls a GetTypes or GetExportTypes in their code
